projects/plugins/jetpack/to-test.md

```diff
@@ -10,9 +10,58 @@
   - Or add the following to something like a code snippet plugin: `add_filter( 'jetpack_blocks_variation', function () { return 'beta'; } );`
 	- To test Breve further in the document please enable the feature with the following snippet: `add_filter( 'breve_enabled', '__return_true' );`
 
-### Your Feature
+### Blocks
 
-Add testing instructions below.
+#### Breve
+
+- Make sure you are on the free plan.
+- Check that Breve is not loaded.
+- Add a Jetpack plan.
+- Check that Breve is enabled.
+- Hide the Jetpack AI Assistant block.
+- Check that Breve is disabled.
+- Unhide the block.
+- Check that Breve is re-enabled without a need to reload.
+
+#### Don't attempt to use blocks that aren't available
+
+It's possible to globally disable blocks for the entire site. This impacts blocks' availability in the editor, but also any other place on the site that relies on them or their code. The old classic slideshow is a good example of that: when used with the AMP plugin, it attempts to display a Slideshow block instead of the classic slideshow in AMP views. This has been fixed, but please also feel free to do some exploratory testing anywhere else you suspect this could be an issue.
+
+- Make sure the shortcodes module is active.
+- Make sure the AMP plugin is active.
+- In Media > Add New, upload a few images.
+- In Posts > Add New, add a classic block.
+- With the media modal, create a gallery, add the images.
+- Pick the slideshow gallery type.
+- Publish the post.
+- Visit the post.
+- In the admin menu bar, click on the AMP menu item to view the AMP version of the post.
+- The slideshow block is displayed properly.
+- Now go to Jetpack > Settings > Writing
+- Disable the Jetpack Blocks
+- Refresh the AMP page.
+- You should see no Fatal errors. The block may not be displayed properly depending on your AMP settings on the site though.
+
+### Fedivrerse creator meta tag from Mastodon connection
+
+- Go to Jetpack > Settings > Sharing
+- Enable Jetpack Social, and connect your site to a test Mastodon account.
+- Go to Posts > Add New.
+- Write a post and publish it. In the post sidebar, ensure the post is shared to Mastodon.
+- Check the source of the post on the frontend: you should see this: .
+- Disable Jetpack Social.
+- Ensure the meta tag is gone, and that there are no errors on your site.
+Enable Jetpack Social again.
```

Comment:
Yes, it could have been, but it will render somewhat correctly anyway, not breaking the list:
![image](https://github.com/user-attachments/assets/865775ae-fede-4c0c-a349-6948987fc9a5)